spec/lib/happy_hood/slack/client_spec.rb

```diff
@@ -45,9 +46,9 @@
         expect(mock_slack_client).to have_received(:chat_postMessage).with(a_hash_including(
           text: a_string_including(
             "Schitt's Creek",
-            "#{1.day.ago.strftime(DifferenceRenderer::SHORT_DATE_FORMAT)}: $5.00",
```

Comment:
We should probably start to decouple the actual message content from the slack client test but not a blocker, just a thought

Reply:
That's a good idea, tbh. Will do this in a follow up PR!